### PR TITLE
Load scripts from bower_components instead of cdn

### LINKED ISSUE #30: Change the source of the scripts loaded from cdn to bower components.
Use the bower components instead of scripts loaded through cdn.

### PR DESCRIPTION
**Pull request by therohitramesh**
Fixes #30

### CODE REVIEW

File: views/index.hbs [hunk 1]
```diff
@@ -29,10 +29,10 @@
           <div class="one column row">   
           </div> 
           <div class="one column row">
-              <div class="center aligned sixteen wide column"><button class="ui purple basic button massive">Enter in ChatRoom</button></div>
+              <div class="center aligned sixteen wide column"><button onclick="location.href = '/chat';" class="ui purple basic button massive">Enter in ChatRoom</button></div>
```
Comment (sumedh123) [inline views/index.hbs]: @therohitramesh Please do a rebase.

Comment (sumedh123) [reply views/index.hbs]: Also provide deployment.

File: index.js [hunk 2]
```diff
@@ -9,6 +9,7 @@ require('./socketserver.js')(io, app);
 
 app.set('view engine', 'hbs');
 app.use(express.static(__dirname + '/public'));
+app.use(express.static(__dirname + '/bower_components'));
```
Comment (sumedh123) [inline index.js]: Incorrect. Refer to https://expressjs.com/en/starter/static-files.html

**Comment by therohitramesh** [comment]
@sumedh123 please check. Deployment link:
https://omeclone-therohitramesh.herokuapp.com/

File: index.js (hunk 2 above)
Comment (sumedh123) [inline index.js]: Instead of this use ```app.use('bower_components', ...);```

File: package.json [hunk 3]
```diff
@@ -4,7 +4,8 @@
   "description": "Clone of omegle.com",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "postinstall": "node node_modules/bower/bin/bower install"
```
Comment (sumedh123) [inline package.json]: ```node_modules/bower/bin/bower install``` only

**Comment by therohitramesh** [comment]
@sumedh123 Made the changes. Check now:
https://omeclone-therohitramesh.herokuapp.com/

**Comment by sumedh123** [comment]
@therohitramesh Resolve conflicts.

File: package.json [hunk 4]
```diff
@@ -4,7 +4,7 @@
   "description": "Clone of omegle.com",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "postinstall": "node node_modules/bower/bin/bower install"
```
Comment (sumedh123) [inline package.json]: can you please change this?
```"postinstall": "node_modules/bower/bin/bower install"```

File: views/index.hbs [hunk 5]
```diff
@@ -22,17 +22,17 @@
...
           <div class="one column row">
               <div class="center aligned sixteen wide column"><a href="/chat"><button class="ui purple basic button massive">Enter in ChatRoom</button></a></div>
           </div>
       </div>
  </body>
- <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
- <script src="https://cdn.jsdelivr.net/npm/semantic-ui@2.2.13/dist/semantic.min.js"></script>
- </html> 
\ No newline at end of file
+ <script src="/bower_components/jquery/dist/jquery.min.js"></script>
```
Comment (sumedh123) [inline views/index.hbs]: comment this out. or just remove.

File: package.json (hunk 4 above)
Comment (sumedh123) [reply package.json]: Also why did you remove the test script. keep it as it was. we ll add tests later.

**Comment by therohitramesh** [comment]
@sumedh123 Check now. https://omeclone-therohitramesh.herokuapp.com/

**Comment by therohitramesh** [comment]
@sumedh123 https://omeclone-therohitramesh.herokuapp.com/

**Comment by sumedh123** [comment]
Good work.

**Pull request merged**